vector/src/main/java/im/vector/app/features/settings/notifications/VectorSettingsNotificationPreferenceFragment.kt

```diff
@@ -117,6 +123,9 @@ class VectorSettingsNotificationPreferenceFragment :
                         }
                         findPreference<VectorPreference>(VectorPreferences.SETTINGS_NOTIFICATION_METHOD_KEY)
                                 ?.summary = unifiedPushHelper.getCurrentDistributorName()
+                        lifecycleScope.launch {
+                            pushersManager.togglePusherForCurrentSession(true)
```

Comment:
You need a try catch, it will crash if there is for instance no network.

Reply:
Agree, ideally a `runCatching` may be added inside the `PushersManager.togglePusherForCurrentSession()` method. And failure may be prompted to the user.